models/models.js

```diff
@@ -6,9 +6,18 @@ exports.extractTopics = () => {
 };
 
 exports.extractArticleById = (article_id) => {
-    return db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
-    .then(({rows}) => {
-        if (rows.length) return rows[0];
+    const articlePromise = db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id]);
+    const commentsPromise = db.query(`SELECT * FROM comments WHERE article_id = $1;`, [article_id]);
+
```

Comment:
this works, but shouldn't need to do this with two queries. Take a look at the SQL notes for how to do this with just one

Reply:
Done! And still reeling in shock from the fact that it worked on the second attempt after I managed to find the relevant section in the notes